sources/install.sh

```diff
@@ -960,6 +960,16 @@ function wireshark_sources() {
   wireshark.tar.xz
 }
 
+function trilium() {
+  colorecho "[EXEGOL] Installing trilium"
+  wget https://github.com/zadam/trilium/releases/download/v0.43.1/trilium-linux-x64-server-0.43.1.tar.xz
```

Comment:
Can you make sure to pull the latest release ?
You can try the hacks we have in the functions `ysoseral_net` or `pspy`